content/spin/cache.md

```diff
@@ -0,0 +1,64 @@
+title = "Spin Cache"
```

Comment:
Suggest either "The Spin Registry Cache" or "Spin Caches" depending on how you see this page evolving.

Reply:
Gone with The Spin Registry Cache.